tests/e2e-legacy/getPackageTags.mjs

```diff
@@ -0,0 +1,5 @@
+export const getPackageTags = (packages) =>
+  packages
+    .map((name) => name.replace("@aws-sdk/client-", ""))
+    .map((name) => name.replace("-"))
```

Comment:
this is going to replace `'-'` with `'undefined'`, no?

Reply:
Yes.

```console
$ node    
Welcome to Node.js v18.20.2.
Type ".help" for more information.
> "-".replace("-")
'undefined'
```

It's from the existing code: https://github.com/aws/aws-sdk-js-v3/blob/021f2181d3fa7d3a2760c97b66ab76f68c70e2f6/tests/e2e-legacy/index.js#L26-L29

I explicitly added `.replace("-", "")` though in [`49e686d` (#6538)](https://github.com/aws/aws-sdk-js-v3/pull/6538/commits/49e686d11a00430db02c7b8c4d3288409aa30260)